### DISCUSSION TITLE
When using pre-trained models

### DISCUSSION THREAD

**Discussion by phharley17** [victorca25/traiNNer#51] [Q&A]
If I am going to train a model to be with x2 scale, can I use a x4 scale pre-trained model?

I thought I read or heard somewhere that the pre trained model must be of the same scale as your training settings.

If the above is true, is it possible to train a model  without the pre-trained model as base? I actually haven't tested anything, just curious before I do.

**Comment by victorca25** [comment]
Hello! In the case of ESRGAN models, yes, it is possible to use a 4x model as pretrained for a 2x model, however there are a few layers (the upscaling layers) that don't match and will be trained from scratch.

It will work perfectly fine, but you won't be able to interpolate the resulting model with other 2x models. It's not too much of an issue, since there aren't as many 2x models available, but if there is one that is close to what you mean to train, it will be better to use it as pretrained and later you could interpolate with it to control the results.

You can also train without a pretrained model, but it will be entirely trained from scratch, so similarly, it won't be able to interpolate with any other model. Also, it will take a bit longer to fully train.
